tasks/smoke-test/playwright-fixtures/devServer.fixture.ts

```diff
@@ -47,50 +49,56 @@ const test = base.extend<any, DevServerFixtures>({
         )
       }
 
-      const isServerAlreadyUp = await isPortReachable(webServerPort, {
-        timeout: 5000,
+      // @MARK: On second run, Web server is off, but api server is on. Why?!
```

Comment:
Is this change still needed with @jtoar's CI fix?
Or have we at least gotten a better understanding of this, so we can have a more educating comment here?

Reply:
Yeah this was for me during debugging. Removed it now.